app/dashboard/templates/ios.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    {% include 'shared/head.html' %}
+    {% include 'shared/cards.html' %}
+</head>
+<body class="interior {{active}}">
+    <div class="{{active}}-hero__container">
+        <div class="container-fluid header dash">
+            {% include 'shared/nav_internal.html' %}
+        </div>
+        <div class="row ml-5 hidden-lg">
+            <div class="col-6 header-line">
+                One Swipe Away From Your Next Bounty.
+                <div class="row no-gutter">
+            <div class="col-2" style="background: #fff; height:7px; margin:30px 0;">
+                </div>
+                </div>
+                <h3>
+                    Browse funded work in open source software from anywhere, at anytime.
+                </h3>
+                <a href="#">
+                    <img class="iphone" src="/static/v2/images/ios/appstore.png"/>
+                </a>
+            </div>
+            <div class="col-6 header-img">
+                <img src="/static/v2/images/ios/iPhone.png">
+            </div>
+            <div class="col-6 subheader-line"></div>
+        </div>
+        <div class="row visible-md-down">
+            <div class="col-12 header-line">
+                One Swipe Away From Your Next Bounty.
+                <div class="row no-gutter">
+                    <div class="col-2" style="background: #fff; height:7px; margin:30px 0;"></div>
+                </div>
+                <h3>
+                    Browse funded work in open source software from anywhere, at anytime.
+                </h3>
+                <a href="#">
+                    <img class="iphone" src="/static/v2/images/ios/appstore.png"/>
+                </a>
+            </div>
+        </div>
+<!--         <div class="row visible-md-down">
+            <div class="col-6 header-img">
+                <img src="/static/v2/images/ios/iPhone.png">
+            </div>
+        </div> -->
+    </div>
+     <div class="row">
+        <div class="col-2"></div>
+        <div class="col-8 video__wrapper">
+            <video class="video">
+                <source src="https://www.w3schools.com/html/mov_bbb.mp4" type="video/mp4">
+                <source src="https://www.w3schools.com/html/mov_bbb.ogg" type="video/ogg">
+            </video>
+            <div class="playpause"></div>
+        </div>
+        <div class="col-2"></div>
+    </div>
+    
+    <div class="row tag-line">
+        <div class="col-12">
+            <h3 class="text-center">It's Easy As 1-2-3 🎶</h3>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-md-1"></div>
+        <div class="col-md-10">
+            <div class="row">
+                <div class="col-md-4" style="">
+                    <div class="text-center">
+                        <div class="flat-card">
```

Comment:
We already have cards set up [here](https://github.com/gitcoinco/web/blob/master/app/dashboard/templates/toolbox.html#L56). Consider reusing them, we could improve them as well ? Just a thought